src/test/java/ApiHandlersConnectionMockitoTests/ApiHandlersConnectionMockitoTest.java

```diff
@@ -17,30 +17,30 @@
 
 
 @ExtendWith(MockitoExtension.class)
-public class ApiHandlersConnectionMockitoTests {
+public class ApiHandlersConnectionMockitoTest {
 
-    private final String URLProficences = "https://www.dnd5eapi.co/api/races/";
+    public static final String URLProficences = "https://www.dnd5eapi.co/api/races/";
 
     @Mock
-    private HttpResponse<String> response;
+    public  HttpResponse<String> response;
     @Mock
-    private HttpClient client;
+    public  HttpClient client;
```

Comment:
why public?

Reply:
Because I forgot when I was trying to make it in separate package, but it didnt worked. Of course corrected!